images/cosign/README.md

```diff
@@ -10,13 +10,122 @@
 
 ---
 <!--monopod:end-->
+Minimalist Wolfi-based Cosign images for signing and verifying images using Sigstore.
 
-Minimal image with cosign binary. **EXPERIMENTAL**
+- [Documentation](https://edu.chainguard.dev/chainguard/chainguard-images/reference/cosign)
+- [Provenance Information](https://edu.chainguard.dev/chainguard/chainguard-images/reference/cosign/provenance_info/)
+<!-- TODO: add Getting Started Guide - [Getting Started Guide](https://edu.chainguard.dev/chainguard/chainguard-images/reference/cosign/getting-started-cosign/) -->
 
-## Get It!
+## Image Variants
 
-The image is available on `cgr.dev`:
+Our `latest` tag uses the most recent build of the [Wolfi Cosign](https://github.com/wolfi-dev/os/blob/main/cosign.yaml) package. The following tagged variant is available without authentication:
 
+- `latest`: This is an image for running `cosign` commands. It does not include a shell or other applications.
+
+Starting in April, accessing older tags will require authentication.
+
+### Cosign Version
+This will automatically pull the image to your local system and execute the command `cosign version`:
+
+```shell
+docker run --rm cgr.dev/chainguard/cosign version
+```
+
+You should see output similar to this:
+
+```
+  ______   ______        _______. __    _______ .__   __.
+ /      | /  __  \      /       ||  |  /  _____||  \ |  |
+|  ,----'|  |  |  |    |   (----`|  | |  |  __  |   \|  |
+|  |     |  |  |  |     \   \    |  | |  | |_ | |  . `  |
+|  `----.|  `--'  | .----)   |   |  | |  |__| | |  |\   |
+ \______| \______/  |_______/    |__|  \______| |__| \__|
+cosign: A tool for Container Signing, Verification and Storage in an OCI registry.
+
+GitVersion:    2.0.0
+GitCommit:     unknown
+GitTreeState:  unknown
+BuildDate:     unknown
+GoVersion:     go1.20.1
+Compiler:      gc
```

Comment:
It may be worth collapsing this with an elision, so that we don't have to necessarily manually update the README. Like this:

```
  ______   ______        _______. __    _______ .__   __.
 /      | /  __  \      /       ||  |  /  _____||  \ |  |
|  ,----'|  |  |  |    |   (----`|  | |  |  __  |   \|  |
|  |     |  |  |  |     \   \    |  | |  | |_ | |  . `  |
|  `----.|  `--'  | .----)   |   |  | |  |__| | |  |\   |
 \______| \______/  |_______/    |__|  \______| |__| \__|
cosign: A tool for Container Signing, Verification and Storage in an OCI registry.

...
Platform:      linux/arm64
```

Reply:
Done!